src/callbacks_stage/subcell_bounds_check.jl

```diff
@@ -8,9 +8,14 @@
 """
     BoundsCheckCallback(; output_directory="out", save_errors=false, interval=1)
 
-Bounds checking routine for [`SubcellLimiterIDP`](@ref). Applied as a stage callback for SSPRK
-methods. If `save_errors` is `true`, the resulting deviations are saved in
-`output_directory/deviations.txt` for every `interval` time steps.
+Subcell limiting techniques with [`SubcellLimiterIDP`](@ref) are constructed to adhere certain
+local or global bounds. To make sure that these bounds are actually met, this callback calculates
+the maximum deviation from the bounds. The maximum deviation per applied bound is printed to
+the screen at the end of the simulation.
+Additionally, for more insights, the occurring errors can be exported with `save_errors==true`
+every `interval` time steps during the simulation. Then, the maximum deviations since the last
+export are saved in "`output_directory`/deviations.txt".
+It has to be implied as a stage callback for SSPRK.
```

Comment:
```suggestion
The `BoundsCheckCallback` has to be applied as a stage callback for the SSPRK time integration scheme.
```

Reply:
It seems like you resolved this convo without actually applying the suggestion. Was this intended?

Reply:
Ah got it